### ISSUE TITLE
dead or broken links

### ISSUE THREAD

**Issue by pw374** [ocaml/v2.ocaml.org#605]
Apparently, there are a number of dead links that need to be fixed.

see https://github.com/ocaml/ocaml.org/pull/604#issuecomment-68348109

**Comment by amirmc** [comment]
I had a quick look and it seems that many of those are within the tutorials section (which still needs to be reworked) and within that, the ocamlbuild tutorial. 

It also looks like there's one broken link on the install page, which should be fixed first. 

Best wishes,
Amir

## 

sent via mobile

**Comment by Teemperor** [comment]
Note that i just ran grep and looked for http:// prefixes. All the relative links and https links are therefore unchecked, so for example is the "latest" link on this page [1] dead but not on the list.

[1] https://ocaml.org/releases/

**Comment by amirmc** [comment]
Thanks for mentioning that. I recall we have an outstanding issue to do with link checking (#111), which relates to this.

**Comment by Teemperor** [comment]
~~Linkchecker is lacking support for reporting custom HTTP error codes (such as 403) as failures.~~ If you don't mind the additional traffic, I could just ~~patch that into linkchecker an~~ run Linkchecker once a week over ocaml.org like that: http://teemperor.de/pub/ocaml.html

**Comment by agarwal** [comment]
> i just ran grep and looked for http:// prefixes

Ah, this probably explains #607, so I'll revert that one URL.

**Comment by agarwal** [comment]
Is this issue actually different than #111? If not, please close.

**Comment by pw374** [comment]
To me, in #111 we're considering an automatic link checker, while here, it's more about fixing a list of known broken links, but I wouldn't mind closing this one and continuing this discussion in #111.

**Comment by agarwal** [comment]
I'll close since no specific list of links is actually stated. The discussion has turned into first determining a way to get that list, which is thus #111.

**Comment by pw374** [comment]
The list was http://pastebin.com/BrTnx1y8 (from https://github.com/ocaml/ocaml.org/pull/604#issuecomment-68348109) but perhaps they were all fixed by https://github.com/ocaml/ocaml.org/pull/606 ?

**Comment by agarwal** [comment]
I see. @Teemperor can you please let us know.

**Comment by Teemperor** [comment]
@agarwal Yes, i overlooked the fact that the 'dir' in the url was actually just a reference, thanks for fixing that!

And #606 only fixed a part of the broken links. These are the remaining ones: http://teemperor.de/pub/ocaml.html
